byterun/minor_gc.c

```diff
@@ -550,12 +617,28 @@ void caml_empty_minor_heap_domain (struct domain* domain)
     oldify_mopup (&st);
     caml_ev_end("minor_gc/promote");
 
+    caml_ev_begin("minor_gc/ephemerons");
+    for (re = remembered_set->ephe_ref.base;
+         re < remembered_set->ephe_ref.ptr; re++) {
+      value* key = &Op_val(re->ephe)[re->offset];
```

Comment:
This might not be a key: what if `re->offset == CAML_EPHE_DATA_OFFSET` ?

Reply:
Handled this explicitly for clarity: https://github.com/ocamllabs/ocaml-multicore/blob/6578f85cec65336c5eeec699f96ade0bb6a734c6/byterun/minor_gc.c#L640-L644